jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -47,6 +47,7 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
@adriancole Objects is available on Java 7 https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html